docs/voice/intent-recognition/supported-languages.md

```diff
@@ -5,7 +5,7 @@ title: "Supported languages"
 import languages from '!!yaml-loader!../../../intents/languages.yaml';
 import intents from '!!yaml-loader!../../../intents/intents.yaml';
 
-If you don't see your language below, [help us translate!](contributing)
+If you don't see your language below, [help us translate!](../contributing)
```

Comment:
```suggestion
If you don't see your language below, [help us translate!](voice/intent-recognition/contributing)
```

Reply:
Thanks! But why does [this work](https://github.com/home-assistant/developers.home-assistant/blob/147030afd168c0e2f7a8add8c78f3b70a2fc2a06/docs/voice/intent-recognition/contributing.md?plain=1#L11) and [this doesn't](https://github.com/home-assistant/developers.home-assistant/blame/4520e5747263ff187977ff0cfc1ce71344a81368/docs/voice/intent-recognition/supported-languages.md#L8), though?

Reply:
TBH I do not know, it works fine locally.
Maybe some netlify/cloudflare rules
Full path _should_ always work.